content/01-install/01a-install-client-windows.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Install Client on Windows
+metaTitle: Install Client on Windows
+---
+
+This section outlines how to install the Mantis client on your Windows machine.
+
+Follow these instructions to install the Mantis client or go [here for instructions to install the Mantis wallet](/install/install-wallet).
+
+
+## Requirements
+
+These are the prerequisites for installing Mantis:
+* A Java Virtual Machine (JVM) with version 1.8.x.
+    To check the JVM version, use `java -version`.
+    To install Java, follow [these instructions](https://java.com/en/download/help/download_options.html).
+    (Note that the Mantis team has not tested the client with JVM 1.9 or later versions)
+* 4G of memory (RAM)
+    Additional RAM is needed for the DAG file _if mining_ is enabled.
+* 250GB of disk space to ensure fast sync of the node. You will need more than that for the future growth of the chain. An SSD of at least 500G is recommended for full sync.
+
+
+## Install
+
+To install Mantis, follow these steps:
+
+1. Download the client binary from the Mantis Client page at [MantisClient.io](https://mantisclient.io/):
+![Mantis Client page](../images/1-Download-Client.png).
+
+2. Choose and download the correct installer from the Client quick-start page:
+![Windows Installer](../images/2-Download-Installer.png)
+Choose the Windows 64 bit installer, and note the checksum.
+
+> Remember to run checksum verification on your downloads. Refer to [this section](/first-steps/downloads#checksum) for instructions.
+
+
+```
+mv mantis-3.1.0.zip /home/<user>/bin/
```

Comment:
these commands and all below look more like linux commands (not windows) to me

Reply:
That is true. I suspect that the client would not be installed on Windows, in which case this document should not exist. I need expert guidance in this.

Reply:
well, I do run linux on my machine... but there seems to be a mantis.bat file inside bin/ which, I suspect, is the launcher file for windows client...

Reply:
the "mv", "unzip", "export PATH" are typicall linux commands, not found on usual windows... I would just name it by what the command does for the windows part (move, unpack)

Reply:
still linux commands for windows?